### PR TITLE
Add `EXTRABOOTPARAMS_LINE_OFFSET` option for `EXTRABOOTPARAMS_BOOT_LOCAL`

### PR DESCRIPTION
**Pull request by ggardet**
- Verification run: https://openqa.opensuse.org/tests/3570015

### CODE REVIEW

**Review by Vogtinator — COMMENTED** [review]
This drops use of `$args{linux_boot_entry}`. Is no caller using this?

**Comment by ggardet** [comment]
> This drops use of `$args{linux_boot_entry}`. Is no caller using this?

No, it was not used at all.

**Comment by Vogtinator** [comment]
Ok. While I think this should also be made to work OOTB, it's so fragile that this doesn't work for long in practice.

What's missing is documentation for `LINUX_BOOT_ENTRY` in `variables.md`. IMO the name should be changed though, something like `EXTRABOOTPARAMS_LINE_OFFSET` maybe.

**Comment by ggardet** [comment]
> Ok. While I think this should also be made to work OOTB, it's so fragile that this doesn't work for long in practice.

At least, this can be used for debugging purpose. 

> What's missing is documentation for `LINUX_BOOT_ENTRY` in `variables.md`. IMO the name should be changed though, something like `EXTRABOOTPARAMS_LINE_OFFSET` maybe.

Yes, I failed to find a better name but `EXTRABOOTPARAMS_LINE_OFFSET` LGTM.
I will add the doc part.
Thanks for the review.

File: lib/opensusebasetest.pm [hunk 1]
```diff
@@ -668,14 +668,14 @@ sub handle_grub {
...
-    my $linux_boot_entry = $args{linux_boot_entry} // (is_sle('15+') ? 15 : 14);
-    $linux_boot_entry = $linux_boot_entry - 1 if is_aarch64;    # poo#100500
 
     # On Xen PV and svirt we don't see a Grub menu
     # If KEEP_GRUB_TIMEOUT is defined it means that GRUB menu will appear only for one second
     return if (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux') && is_svirt || check_var('KEEP_GRUB_TIMEOUT', '1'));
     $self->wait_grub(bootloader_time => $bootloader_time, in_grub => $in_grub);
     if (my $boot_params = get_var('EXTRABOOTPARAMS_BOOT_LOCAL')) {
+        my $linux_boot_entry = get_var('EXTRABOOTPARAMS_LINE_OFFSET') // (is_sle('15+') ? 15 : 14);
+        $linux_boot_entry = $linux_boot_entry - 1 if is_aarch64 && !get_var('EXTRABOOTPARAMS_LINE_OFFSET');    # poo#100500
```
Comment (Martchus) [inline lib/opensusebasetest.pm:678]:
```suggestion
        $linux_boot_entry -= 1 if is_aarch64 && !get_var('EXTRABOOTPARAMS_LINE_OFFSET');    # poo#100500
```

```suggestion
        --$linux_boot_entry if is_aarch64 && !get_var('EXTRABOOTPARAMS_LINE_OFFSET');    # poo#100500
```

Comment (ggardet) [reply lib/opensusebasetest.pm]: Merged.

**Review by Martchus — APPROVED** [review]

**Pull request merged**